tests/e2e/e2e_test.go

```diff
@@ -99,6 +104,22 @@ func TestE2e(t *testing.T) {
 		t.Run(testname, func(t *testing.T) {
 			// Default 2 minute timeout
 			onGiteaRun(t, func(*testing.T, *url.URL) {
+				err := db.Iterate(context.Background(), nil, func(ctx context.Context, user *user_model.User) error {
```

Comment:
I think we could remove all avatars from the `user.yml` (make them use default avatar), then no need to use this trick.

Reply:
This is a good call! I implemented it here: https://github.com/go-gitea/gitea/pull/32529/commits/b05e55c2abe3685e90e410e75ae72f4d50be7931

Reply:
It seems this caused some tests to fail since they were generating avatars on demand. I updated the fixtures in https://github.com/go-gitea/gitea/pull/32529/commits/bef5f300a4f62b86d89ad3c0a049c564209635cf to prevent this from happening, except for user 2, which is explicitly tested in some cases.